park_api/cities/Wiesbaden.py

```diff
@@ -0,0 +1,65 @@
+from bs4 import BeautifulSoup
+from park_api.util import convert_date
+from park_api.geodata import GeoData
+
+# This loads the geodata for this city if <city>.geojson exists in the same directory as this file.
+# No need to remove this if there's no geodata (yet), everything will still work.
```

Comment:
Can we remove the sample city comments?

Reply:
Fixed in https://github.com/offenesdresden/ParkAPI/pull/181/commits/66e42c743c188e8e6a3e5fb964b985aec274b16e.